sleap/io/format/deeplabcut.py

```diff
@@ -69,7 +69,7 @@ def does_write(self) -> bool:
     def read(
         cls, file: FileHandle, full_video: Optional[Video] = None, *args, **kwargs,
     ) -> Labels:
-        return Labels(labeled_frames=cls.read_frames(file, full_video, *args, **kwargs))
```

Comment:
Are we sure that skeleton doesn't ever get passed in here through the `**kwargs`?

Reply:
not sure if I understand the question - the 'read_frames' method does not access args / kwargs

Reply:
but the method read_frames is not reading anything from kwargs, so this is a no-op right?

Reply:
It's not that `read_frames` is using the `**kwargs`, it's that it's using `skeleton`. Previously, if `kwargs = {"skeleton": [...]}` in `read()`, that would pass through to the named keyword arg in `read_frames()`. With this change, you cannot pass a skeleton to `read_frames()` through `read()`.

I don't think this is a serious use case, but to ensure it is equivalent to the previous behavior, `read()` should be:

```py
@classmethod
def read(cls, file: FileHandle, full_video: Optional[Video] = None, skeleton: Optional[Skeleton] = None, *args, **kwargs) -> Labels:
    return Labels(labeled_frames=cls.read_frames(file=file, full_video=full_video, skeleton=skeleton))
```

See what I'm saying?

Reply:
Thanks for the explanation - adding them back